Enterprise-CMCS/macpro-onemac
# PI1.3 26084: Withdraw Package screen and email messaging

Story: https://qmacbis.atlassian.net/browse/OY2-26084
Endpoint: [Click here](https://d2pw7kqcao5lf0.cloudfront.net/)

### Details
After seeing the demo of the RAI Response Withdraw process, our Business Owners wanted to change some of the language around Package Withdrawal to make it more specific to the user and better differentiate it from RAI Response Withdraw.

### Changes
- Followed the ONEMAC_LABEL[parentType] pattern used in the Type field on the Withdraw form as the <Package Type> for this story
- Updated Introductory text on Withdraw Package forms (now includes the package type)
- Updated submit confirmation heading and text for Withdraw Package form.
- Updated email receipt sent to state user upon successful Withdrawal Request submission.
- ALSO fixed two warnings/errors in validateUser and getMyPackages (territoryList === [] no longer works)
- ALSO added some TE seed data so the first build TE tests don't fail.

### Test Plan
1. Login as a state submitter user for whom you have email access
2. Navigate to Dashboard
3. For any package type that can be withdrawn (all but TEs):
 a. Find a package of that type with "Withdraw Package" as an available action
 b. Verify the Intro text on the withdraw form matches the AC
 c. Complete form and click Submit
 d. Verify the Confirmation Heading and text match the AC
 e. Confirm the Submit
 f. Verify the email matches the AC.
